package.json

```diff
@@ -30,7 +30,7 @@
 		"node": ">=16"
 	},
 	"peerDependencies": {
-		"eslint": ">=6.0.0"
+		"eslint": ">=8.53.0"
```

Comment:
Shouldn't this also be 8.0.0 as the minimum supported version

Reply:
Does it not work if you set it to 8.0.0? The commit message says the dependency is just 8+...

Reply:
Fine.